src/fair/item.py

```diff
@@ -150,3 +150,34 @@ def __init__(
         """
         super().__init__("schedule", features, values, index, capacity)
         self.category = category
+
+
+def sub_schedule(bundles: list[list[ScheduleItem]]):
+    """Given a list of bundles, create a new sub schedule considering the items in the union of all bundles
+    Capacities of the new schedule are determined by the sum of the capacities of the items in all bundles.
+
+    This function is necessary to compute PMMS metric. Creates a schedule with the items currently owned by two agents, in order to solve subproblem.
+
+    Args:
+        bundles (list[list[ScheduleItem]]): List of Items from class BaseItem
+
+    Returns:
+        new_schedule (list[ScheduleItem]): Items from class BaseItem, new reduced schedule
+        course_strings (list[str]): List of course strings of the new schedule
+        course (type[Course]): Course instance of the new schedule
+    """
+    sub_schedule = [item for bundle in bundles for item in bundle]
+    set_sub_schedule = sorted(list(set(sub_schedule)), key=lambda item: item.values[0])
+
+    features = sub_schedule[0].features
+    course_strings = sorted([item.values[0] for item in set_sub_schedule])
+
+    new_schedule = []
+    for i, item in enumerate(set_sub_schedule):
+        new_capacity = [
+            new_item.capacity for new_item in sub_schedule if new_item == item
+        ]
+        new_schedule.append(
+            ScheduleItem(features, item.values, index=i, capacity=sum(new_capacity))
+        )
+    return new_schedule, course_strings
```

Comment:
`course_strings` does not belong in this method. It should be defined in `pairwise_maximin_share` after `sub_schedule` is called.

Reply:
Good catch! I don't know what it was doing there, just changed it.